app/src/main/java/rick_and_morty/ui/characters/composables/CharacterList.kt

```diff
@@ -0,0 +1,84 @@
+import androidx.compose.foundation.layout.Arrangement
+import androidx.compose.foundation.lazy.LazyColumn
+import androidx.compose.foundation.lazy.items
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.LaunchedEffect
+import androidx.compose.runtime.collectAsState
+import androidx.compose.ui.unit.dp
+import androidx.lifecycle.viewmodel.compose.viewModel
+import rick_and_morty.ui.characters.CharactersViewModel
+
+@Composable
+fun CharacterList() {
+    val charactersViewModel = viewModel(modelClass = CharactersViewModel::class.java)
```

Comment:
this can be moved to a parameter with a default value, making it possible to provide a test double in test code.

Reply:
move to parameter and make it default

Reply:
Done